changelog.md

```diff
@@ -17,6 +17,7 @@
 - [DCOS-51375](https://jira.mesosphere.com/browse/DCOS-51375) - Fixed an issue where deployment cancellation could leak instances.
 
 - [DCOS_OSS-5211](https://jira.mesosphere.com/browse/DCOS_OSS-5211) - The initial support for volume profiles would match disk resources with a profile, even if no profile was required. This behavior has been adjusted so that disk resources with profiles are only used when those profiles are required, and are not used if the service for which we are matching offers does not require a disk with that profile.
+- [MARATHON-8631](https://jira.mesosphere.com/browse/MARATHON-8631) - When using DC/OS Storage Service (DSS), users can provision storage volumes as disk resources tagged with a profile name. Marathon allows users to create services that make use of disks tagged with these profiles. The type of disk resource in Mesos will be `Mount` in this case, as this is the only disk type DSS supports. Therefore, Marathon should default to disk type `Mount` instead of `Root`, when a profileName is given for a persistent volume configuration.
```

Comment:
I'm curious if we should state this without mentioning DC/OS?   Can't we state what it is DSS does (perhaps still using it as an example) and how Marathon plays into that?    I would love to see more generic or Mesos specific language here... DCOS specific language in DCOS dcos.

Reply:
I realize this is change logs

Reply:
That makes sense to me, thanks for the suggestion. I'll update the changelog entry providing a link to the DSS docs. I'm afraid it's not too helpful to stay in a pure Mesos world, since there is no other tool that creates/manages volumes.